### PR TITLE
Create a stub of the kkarpala elmira file

### PR DESCRIPTION
**Pull request by kkarpala**
This file will add things to do in Elmira, ON

- [ ] Add places to eat
- [ ] Add things to do
- [ ] Add things to avoid

### CODE REVIEW

**Comment by kkarpala** [comment]
@nathyga what do you like to eat?

File: kkarpala-elmira.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## Elmira, Ontario, Canada
```
Comment (francisvicta) [inline kkarpala-elmira.md]: Are there tigers in Elmira?

File: kkarpala-elmira.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+### Places to eat
+
+### Attractions nearby
+
+### Culture
```
Comment (kkarpala) [inline kkarpala-elmira.md]: Mennonites and rural culture

File: kkarpala-elmira.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+## Elmira, Ontario, Canada
+
+### Places to eat
```
Comment (nathyga) [inline kkarpala-elmira.md]: Is there an Italian restaurant in Elmira, ON?

File: kkarpala-elmira.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+
+### Attractions nearby
```
Comment (kkarpala) [inline kkarpala-elmira.md]: St Jacobs Farmers Market, Elora Gorge, West Montrose Kissing Bridge, Elmira Maple Syrup festival

File: kkarpala-elmira.md (hunk 1 above)
Comment (kkarpala) [inline kkarpala-elmira.md]: Sadly no tigers

**Pull request merged**